vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -33,9 +33,10 @@ import (
 )
 
 // DefaultRecommenderName denotes the current recommender name as the "default" one.
-const DefaultRecommenderName = "default"
+const defaultRecommenderName = "default"
```

Comment:
I think we can drop this const, set `"default"` as the default value of `recommenderName` flag